src/main/java/org/skriptlang/skript/bukkit/recipes/elements/ExprRecipeResult.java

```diff
@@ -0,0 +1,88 @@
+package org.skriptlang.skript.bukkit.recipes.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.EventValueExpression;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.RecipeUtils.RegisterRecipeEvent;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.Recipe;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Recipe Result")
+@Description("The result item for a recipe")
+@Examples({
+	"register a new shaped recipe with the key \"my_recipe\":",
+		"\tset the recipe ingredients of 1st row to diamond, air and diamond",
+		"\tset the recipe result to diamond sword named \"Chosen One\""
+})
+@Since("INSERT VERSION")
+public class ExprRecipeResult extends PropertyExpression<Recipe, ItemStack> {
+
+	static {
+		Skript.registerExpression(ExprRecipeResult.class, ItemStack.class, ExpressionType.PROPERTY,
+			"[the] recipe result[ing] [item] [of %-recipes%]");
```

Comment:
shouldn't be nullable

Reply:
I think the idea here is that the expression can be used in a recipe section, and therefor no need to specify the recipe?!?!?

Reply:
Yeah, but that should be done via a default value: ``[of %recipe%]`` that's filled in by the section's event-recipe

Reply:
oh right, duh... my bad.

Reply:
You should wrap them in a mutable wrapper, then

Reply:
How would I go about doing this? Is there a reference by chance?